src/classes/HiFiAudioAPIData.ts

```diff
@@ -280,7 +464,7 @@ export class ReceivedHiFiAudioAPIData extends HiFiAudioAPIData {
      * ❌ The client never sends `volumeDecibels` data to the server.
      * ✔ The server sends `volumeDecibels` data to all clients connected to a server during "peer updates".
      */
-    constructor(params: { providedUserID?: string, hashedVisitID?: string, volumeDecibels?: number, position?: Point3D, orientationEuler?: OrientationEuler3D, orientationQuat?: OrientationQuat3D, hiFiGain?: number } = {}) {
+    constructor(params: { providedUserID?: string, hashedVisitID?: string, volumeDecibels?: number, position?: Point3D, orientation?: OrientationQuat3D, hiFiGain?: number } = {}) {
```

Comment:
this one seems like it might be confusing for developers.

i vote keep the `orientationQuat` name in here so folks aren't confused about whether or not the received orientation is quat vs euler.

this might not be the only place where keeping the old name would be helpful, it's just the one i noticed first